### PR TITLE
dsview: 1.3.1 -> 1.3.2

### LINKED ISSUE #319411: nixpkgs.dsview dsview-1.3.1 ModuleNotFoundError
### Describe the bug
Decoders fail to load due to `ModuleNotFoundError`.

### Steps To Reproduce
1. `nix-env -iA nixpkgs.dsview`
2. `DSView`

### Expected behavior
- Working Module Imports (no `ModuleNotFoundError`)
- Working decoders

### Additional context
```
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.

(process:16417): Gtk-WARNING **: 01:12:13.472: Locale not supported by C library.
	Using the fallback 'C' locale.
Gtk-Message: 01:12:13.472: Failed to load module "xapp-gtk3-module"
DSView: ----------------- version: 1.3.1-----------------
DSView: Qt:6.7.1
DSView: 2024-06-13 01:12:13
DSView: GetAppDataDir:"/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/DSView"
DSView: GetFirmwareDir:"/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/DSView/res"
DSView: GetUserDataDir:"/home/semno/.local/share/DreamSourceLab/DSView"
DSView: GetDecodeScriptDir:"/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders"
sr: lib_main: Init libsigrok.
sr: lib_main: Scan all connected hardware device.
sr: dslogic: Scan DSLogic device...
sr: dslogic: Found new DSLogic device count: 0
sr: dscope: Scan DSCope device...
sr: dscope: Found new DSCope device count: 0
sr: backend: Register hotplug callback.
sr: lib_main: Hotplug thread start!
srd: Failed to load decoder 0-uart: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/0-uart/__init__.py", line 40, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/0-uart/pd.py", line 23, in <module>
    from math import floor, ceil
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder 1-uart: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/1-uart/__init__.py", line 40, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/1-uart/pd.py", line 23, in <module>
    from math import floor, ceil
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder ade77xx: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/ade77xx/__init__.py", line 32, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/ade77xx/pd.py", line 24, in <module>
    import math
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder amulet_ascii: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/amulet_ascii/__init__.py", line 28, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/amulet_ascii/pd.py", line 21, in <module>
    from math import ceil
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder arm_etmv3: import by name failed:ModuleNotFoundError:No module named '_posixsubprocess'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/arm_etmv3/__init__.py", line 25, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/arm_etmv3/pd.py", line 21, in <module>
    import subprocess
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/subprocess.py", line 104, in <module>
    from _posixsubprocess import fork_exec as _fork_exec
ModuleNotFoundError: No module named '_posixsubprocess'

srd: Failed to load decoder arm_itm: import by name failed:ModuleNotFoundError:No module named '_posixsubprocess'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/arm_itm/__init__.py", line 25, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/arm_itm/pd.py", line 22, in <module>
    import subprocess
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/subprocess.py", line 104, in <module>
    from _posixsubprocess import fork_exec as _fork_exec
ModuleNotFoundError: No module named '_posixsubprocess'

srd: Failed to load decoder dcf77: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/dcf77/__init__.py", line 28, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/dcf77/pd.py", line 21, in <module>
    import calendar
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/calendar.py", line 9, in <module>
    import datetime
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/datetime.py", line 12, in <module>
    import math as _math
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder graycode: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/graycode/__init__.py", line 24, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/graycode/pd.py", line 21, in <module>
    import math
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder i2s: import by name failed:ModuleNotFoundError:No module named '_struct'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/i2s/__init__.py", line 29, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/i2s/pd.py", line 22, in <module>
    import struct
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/struct.py", line 13, in <module>
    from _struct import *
ModuleNotFoundError: No module named '_struct'

srd: Failed to load decoder ir_irmp: import by name failed:ModuleNotFoundError:No module named '_ctypes'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/ir_irmp/__init__.py", line 25, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/ir_irmp/pd.py", line 21, in <module>
    from . import irmp_library
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/ir_irmp/irmp_library.py", line 24, in <module>
    import ctypes
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/ctypes/__init__.py", line 8, in <module>
    from _ctypes import Union, Structure, Array
ModuleNotFoundError: No module named '_ctypes'

srd: Failed to load decoder lfast: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/decimal.py", line 3, in <module>
    from _decimal import *
ModuleNotFoundError: No module named '_decimal'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/lfast/__init__.py", line 35, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/lfast/pd.py", line 22, in <module>
    import decimal
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/decimal.py", line 8, in <module>
    from _pydecimal import *
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/_pydecimal.py", line 156, in <module>
    import math as _math
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder modbus: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/modbus/__init__.py", line 28, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/modbus/pd.py", line 22, in <module>
    from math import ceil
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder numbers_and_state: import by name failed:ModuleNotFoundError:No module named '_struct'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/numbers_and_state/__init__.py", line 41, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/numbers_and_state/pd.py", line 43, in <module>
    import struct
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/struct.py", line 13, in <module>
    from _struct import *
ModuleNotFoundError: No module named '_struct'

srd: Failed to load decoder ook_oregon: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/ook_oregon/__init__.py", line 25, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/ook_oregon/pd.py", line 21, in <module>
    import math
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder pjdl: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/pjdl/__init__.py", line 27, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/pjdl/pd.py", line 58, in <module>
    from math import ceil, floor
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder pjon: import by name failed:ModuleNotFoundError:No module named '_struct'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/pjon/__init__.py", line 25, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/pjon/pd.py", line 36, in <module>
    import struct
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/struct.py", line 13, in <module>
    from _struct import *
ModuleNotFoundError: No module named '_struct'

srd: Failed to load decoder sipi: import by name failed:ModuleNotFoundError:No module named 'binascii'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/sipi/__init__.py", line 30, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/sipi/pd.py", line 21, in <module>
    from binascii import crc_hqx
ModuleNotFoundError: No module named 'binascii'

srd: Failed to load decoder swim: import by name failed:ModuleNotFoundError:No module named 'math'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/swim/__init__.py", line 29, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/swim/pd.py", line 21, in <module>
    import math
ModuleNotFoundError: No module named 'math'

srd: Failed to load decoder usb_power_delivery: import by name failed:ModuleNotFoundError:No module named '_struct'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/usb_power_delivery/__init__.py", line 24, in <module>
    from .pd import *
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/usb_power_delivery/pd.py", line 23, in <module>
    import struct
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/struct.py", line 13, in <module>
    from _struct import *
ModuleNotFoundError: No module named '_struct'

srd: Failed to load decoder usb_request: import by name failed:ModuleNotFoundError:No module named '_struct'.
srd: Traceback (most recent call last):
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/usb_request/__init__.py", line 49, in <module>
    from .pd import Decoder
  File "/nix/store/zmbbm0clik21x5yh1932hhx701mzm12p-dsview-1.3.1/share/libsigrokdecode4DSL/decoders/usb_request/pd.py", line 21, in <module>
    import struct
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/struct.py", line 13, in <module>
    from _struct import *
ModuleNotFoundError: No module named '_struct'

DSView: Set default device.
sr: lib_main: Start activating device.
sr: lib_main: Activating device name: "Demo Device".
sr: lib_main: To open device, name:"Demo Device"
sr: lib_main: Activating device end.
DSView: Switch to device "Demo Device" done.
DSView: Update device list.
DSView: Update rate list.
DSView: Update sample count list.
DSView: Protocol not exists! id:1:uart
DSView: SigSession::Close(), stop capture
DSView: Main window closed.
sr: lib_main: Uninit libsigrok.
sr: lib_main: Release current actived device. name:"Demo Device"
sr: backend: Unregister hotplug callback.
sr: lib_main: Hotplug thread end!
DSView: Uninit log.
```

### Notify maintainers
@bachp 
@carlossless 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.0-112-generic, Linux Mint, 21.3 (Virginia), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by bachp**
Use Qt5 for now as Qt6 is not properly supported yet https://github.com/DreamSourceLab/DSView/issues/777#issuecomment-2144118728

## Description of changes

Update to latest version and use Qt5.

Closes #319411

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by NickCao — COMMENTED** [review]
Please modify https://github.com/NixOS/nixpkgs/blob/f397163ac28b774e69d4dee8bd159812a7e0df77/pkgs/top-level/all-packages.nix#L7394 instead for using qt5.

**Review by carlossless — APPROVED** [review]

File: pkgs/applications/science/electronics/dsview/default.nix [hunk 1]
```diff
@@ -1,18 +1,20 @@
 { stdenv, lib, fetchFromGitHub, pkg-config, cmake, wrapQtAppsHook
-, libzip, boost, fftw, qtbase, qtwayland, qtsvg, libusb1
+, libzip, boost, fftw, libusb1, qtbase, qtsvg, qtwayland
 , python3, desktopToDarwinBundle
 }:
 
+
+
```
Comment (Aleksanaa) [inline pkgs/applications/science/electronics/dsview/default.nix:7]: Don't add too many blank lines

**Comment by JohnRTitor** [comment]
Result of `nixpkgs-review pr 319641` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>dsview</li>
  </ul>
</details>

**Review by JohnRTitor — APPROVED** [review]
LGTM, lets merge this one after CI passes

**Pull request merged**